lib/Command/MigrateGoogleTakeout.php

```diff
@@ -190,7 +190,29 @@ protected function migrateFile(File $file): void
         $jsonFile = null;
 
         try {
-            $jsonPath = $path.'.json';
+            /* the JSON file may contain the "supplemental-metadata" string, fully or partially */
+            $nodes = $this->rootFolder->getDirectoryListing();
```

Comment:
I'm somewhat concerned about this since it will enlist the directory for each file - if you've large directories this is likely too slow with O(n^2)

Reply:
You're totally right, I've addressed your comments